htmresearch/encoders/__init__.py

```diff
@@ -22,8 +22,15 @@
 from enum import Enum
 
 
-
+# TODO: this should be deleted (or at least moved to cio_encoder.py)
 class EncoderTypes(Enum):
   """Defines enumerations for the Cortical.io encoder types."""
+
+  # Encode an entire document at a time
   document = "document"
+
+  # Encode a document by encoding individual words and unioning the result
   word = "word"
+
+  # Encode a single token
+  token = "token"
```

Comment:
> Where is this used?

Line 120 of cio_encoder.py (it's implicit in the "else" clause as the third option).